README.md

```diff
@@ -1,5 +1,119 @@
+- [PySpark](https://pypi.org/project/pyspark/)  is required for distributed training or to ingest Spark tables.
+- [Delta](https://pypi.org/project/delta-spark/) is required to ingest Delta tables.
+These libraries are available natively in the [Databricks Runtime for Machine Learning](https://docs.databricks.com/runtime/mlruntime.html).
+
+## Log to the designated MLflow Experiment
+To log recipe runs to a particular MLflow experiment:
+1. Open `profiles/databricks.yaml` or `profiles/local.yaml`, depending on your environment.
+2. Edit (and uncomment, if necessary) the `experiment` section, specifying the name of the
+   desired experiment for logging.
+
+## Development Environment -- Databricks
+[Sync](https://docs.databricks.com/repos.html) this repository with
+[Databricks Repos](https://docs.databricks.com/repos.html) and run the `notebooks/databricks`
+notebook on a Databricks Cluster running version 11.0 or greater of the
+[Databricks Runtime](https://docs.databricks.com/runtime/dbr.html) or the
+[Databricks Runtime for Machine Learning](https://docs.databricks.com/runtime/mlruntime.html)
+with [workspace files support enabled](https://docs.databricks.com/repos.html#work-with-non-notebook-files-in-a-databricks-repo).
+
+**Note**: When making changes to recipes on Databricks,
+it is recommended that you edit files on your local machine and
+use [dbx](https://docs.databricks.com/dev-tools/dbx.html) to sync them to Databricks Repos, as
+demonstrated [here](https://mlflow.org/docs/latest/recipes.html#usage)
+
+**Note**: data profiles display in step cards are not visually compatible with dark theme.
+Please avoid using the dark theme if possible.
+
+### Accessing MLflow recipe Runs
+You can find MLflow Experiments and MLflow Runs created by the recipe on the
+[Databricks ML Experiments page](https://docs.databricks.com/applications/machine-learning/experiments-page.html#experiments).
+
+## Development Environment -- Local machine
+### Jupyter
+
+1. Launch the Jupyter Notebook environment via the `jupyter notebook` command.
+2. Open and run the `notebooks/jupyter.ipynb` notebook in the Jupyter environment.
+
+**Note**: data profiles display in step cards are not visually compatible with dark theme.
+Please avoid using the dark theme if possible.
+
+### Command-Line Interface (CLI)
+
+First, enter the corresponding example root directory and set the profile via environment variable.
+For example, for the regression example project,
+```
+cd regression
+```
+```
+export MLFLOW_RECIPES_PROFILE=local
+```
+
+Then, try running the
+following [MLflow Recipes CLI](https://mlflow.org/docs/latest/cli.html#mlflow-recipes)
+commands to get started.
+Note that the `--step` argument is optional.
+Recipe commands without a `--step` specified act on the entire recipe instead.
+
+Available step names are: `ingest`, `split`, `transform`, `train`, `evaluate` and `register`.
```

Comment:
Should we move this to ReadMe of individual folders since this is a example of all possible recipes. The steps could be different. So having it in the readme along with the example might be useful. What do you think?

Reply:
I'll merge it for now and udpate it in another PR (to unblock bug bash).